### DISCUSSION TITLE
SecurityOnion 2.3 AWS AMI web console (SOC) connection issue

### DISCUSSION THREAD

**Discussion by rmarkscott** [Security-Onion-Solutions/securityonion#3561] [Unsupported Versions]
I just installed the SO 2.3 AWS AMI in Eval mode for testing. I can SSH into it, but I cannot https into the SOC web console. All services are running ok according to so-status. I have added the appropriate allow IPs using so-allow as seen below:

[onion@securityonion-internal ~]$ sudo so-firewall includedhosts analyst
127.0.0.1
64.22.x.x
172.31.16.0/20

Using Safari, https://ipaddress-of-my-aws-ami and get this message that "This connection is not private". Then I can override and still "visit this website". Then I get this message "Safari cannot open https://172.20.16.102 because Safari cannot connect to server https://172.31.20.102. 172.31.20.102 is my SOC. 

Is anyone else using this AMI that can provide some guidance on what to do?

I am using this for an intrusion detection class I teach at my university. At this point, the students will connect as analysts to review the data after I run so-test. This is their first time to use SO.

**Comment by jertel** [comment]
What did you choose during the setup wizard, when it asked how you will connect to the web interface? It lists three options: IP, Hostname, and Other.

**Comment by rmarkscott** [comment]
The IP option. 

Mark Scott, M.S., Assistant Professor, GCIA
Cybersecurity Program Coordinator
Member - The FBI’s InfraGard Program
Freed-Hardeman University | 158 E. Main Street | Henderson, TN 38340
731.989.6779

http://www.fhu.edu/cybersecurity

**Reply by jertel** [reply]
The IP option will generally use a LAN/internal network IP. So when you are accessing the EC2 public IP in your browser, the SO components are redirecting you to the internal IP, which your browser does not have access to connect.

You will need to use the "Other" option and either specify the EC2 Public IP, or enter a hostname for which you will setup DNS and have it route to the EC2 Public IP.

**Comment by rmarkscott** [comment]
I will try that. Thank you for your response. 

Mark Scott, M.S., Assistant Professor, GCIA
Cybersecurity Program Coordinator
Member - The FBI’s InfraGard Program
Freed-Hardeman University | 158 E. Main Street | Henderson, TN 38340
731.989.6779

http://www.fhu.edu/cybersecurity
